nodes/src/nodes/llm_base/resilience.py

```
@@ -0,0 +1,437 @@
+    """
+    if retryable_types is not None:
+        return isinstance(exc, retryable_types)
+    name = type(exc).__name__
+    if name in _DEFAULT_NON_RETRYABLE_NAMES:
+        return False
+    if name in _DEFAULT_RETRYABLE_NAMES:
+        return True
+    # Treat generic connection / timeout errors as retryable.
+    if isinstance(exc, (ConnectionError, TimeoutError, OSError)):
+        return True
+    return False
```

Ruff (0.15.7)

[warning] Return the condition directly

Inline condition

(SIM103)